src/components/Carousel/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useRef } from 'react';
+
+import { Carousel as BootstrapCarousel } from 'bootstrap';
+import classNames from 'classnames';
+
+export const carouselTestIds = {
+  carousel: 'carousel',
+  carouselIndicator: 'carousel-indicator',
+  carouselItem: 'carousel-item',
+};
+
+export interface CarouselProps<T> extends React.HTMLAttributes<HTMLDivElement> {
+  id: string;
+  items: T[];
+}
+
+const Carousel = <T extends React.ReactNode>({ id, className, items, ...props }: CarouselProps<T>) => {
```

Comment:
The requirement mentions that users can horizontally scroll through carousel items. I am not sure if the bootstrap one supports this action.

Reply:
Yup, I have [an issue](https://github.com/rosle/react-nimble-survey/issues/55) specifically for this. Going to implement it later maybe using some other library. Hope it doesn't introduce much changes 🤞